tests/pipelines/test_pipelines_zero_shot_image_classification.py

```diff
@@ -85,6 +85,7 @@ def test_small_model_pt(self):
             [
                 [{"score": 0.333, "label": "a"}, {"score": 0.333, "label": "b"}, {"score": 0.333, "label": "c"}],
                 [{"score": 0.333, "label": "a"}, {"score": 0.333, "label": "c"}, {"score": 0.333, "label": "b"}],
+                [{"score": 0.333, "label": "b"}, {"score": 0.333, "label": "a"}, {"score": 0.333, "label": "c"}],
```

Comment:
@ydshieh I had to add another possible permutation here. I considered it OK as the comment above mentions the order isn't guaranteed.

Reply:
No problem! Thanks for informing me.